Fastlane/shared_lanes.rb

```diff
@@ -117,10 +117,33 @@ def slack_message(message, options = {})
   )
 end
 
+lane :clean_up_release_from_tag do |options|
+  latest_tag = ENV['BITRISE_GIT_TAG']
+
+  latest_release_branch = "release/#{latest_tag}"
+  if `git ls-remote --heads origin #{latest_release_branch}`.empty?
+    puts "Branch #{latest_release_branch} doesn't exist. Nothing to delete."
```

Comment:
Fastlane actually encourages to use their internal "messaging" instead of puts.
```
UI.message "some note"
UI.important "some warning"
UI.success "something positive :)"
```

Reply:
```suggestion
    UI.message "Branch #{latest_release_branch} doesn't exist. Nothing to delete."
```